.murdock

```diff
@@ -187,7 +187,7 @@ _greplist() {
 # get list of all app directories
 get_apps() {
     make -f makefiles/app_dirs.inc.mk info-applications \
-        | $(_greplist $APPS) | sort
+        | $(_greplist $APPS) | sort | grep lz4
```

Comment:
just drop that :wink:

Reply:
I'm not sure Murdock will pass, but I'll do it does the next time (should be fast)

Reply:
you'll still need a `Makefie.ci` - you could use the new `make  create-Makefile.ci` target for that

Reply:
BTW, file scope `APPS="tests/pkg_lz4` would also have worked.